backend/lucene/src/main/java/org/hibernate/search/backend/lucene/work/impl/LuceneStubWorkFactory.java

```diff
@@ -38,6 +38,11 @@ public LuceneStubWorkFactory(MultiTenancyStrategy multiTenancyStrategy) {
 		return multiTenancyStrategy.createDeleteEntryLuceneWork( indexName, tenantId, id );
 	}
 
+	@Override
+	public LuceneIndexWork<?> delete(String indexName, String tenantId) {
```

Comment:
Maybe rename this "deleteAll"? It's a bit dangerous to name it as if it deleted only one entry.

Reply:
This was the other option I had thought. I will change it.